pkg/apis/messaging/v1alpha1/parallel_types.go

```diff
@@ -76,39 +77,21 @@ type ParallelSpec struct {
 
 	// Reply is a Reference to where the result of a case Subscriber gets sent to
 	// when the case does not have a Reply
-	//
-	// You can specify only the following fields of the ObjectReference:
-	//   - Kind
-	//   - APIVersion
-	//   - Name
-	//
-	//  The resource pointed by this ObjectReference must meet the Addressable contract
-	//  with a reference to the Addressable duck type. If the resource does not meet this contract,
-	//  it will be reflected in the Subscription's status.
 	// +optional
-	Reply *corev1.ObjectReference `json:"reply,omitempty"`
+	Reply *v1alpha1.Destination `json:"reply,omitempty"`
 }
 
 type ParallelBranch struct {
 	// Filter is the expression guarding the branch
-	Filter *SubscriberSpec `json:"filter,omitempty"`
```

Comment:
SubscriberSpec has a *string as URI, as opposed to our Destination, which is *apis.URL. Check if this can break...